app/controllers/consultations_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ConsultationsController < ApplicationController
   before_action :set_consultation, only: [:show, :edit, :update, :destroy]
+  before_action :authenticate_user!, only: [:new, :create]
```

Comment:
Technically we should authenticate user only when trying to run the create method though, right? Not new? Cause the consultations#new page is still an intro to what we're doing and only by clicking on the Let's start button should we be creating a consultation on the back-end? That's what I understood, I may be wrong though..

Reply:
yea definitely, I opened the PR a few hours before the call so it's a wee bit behind our convo